packages/webpack-config/package.json

```diff
@@ -31,16 +33,20 @@
     "copy-webpack-plugin": "^5.0.0",
     "css-loader": "^2.1.1",
     "deep-diff": "^1.0.2",
+    "eslint": "^6.0.1",
+    "eslint-config-universe": "^1.0.7",
```

Comment:
We should probably roll a custom eslint config this. Currently I have it disabled by default.

Reply:
That probably needs to happen soon. Currently if you enable the linter option in this webpack config and your project depends on eslint-config-universe, ESLint's lookup algorithm will use your project's copy of the linter config. The more the linter validation can be encapsulated away from other project settings the better, since its job is just to check for basic JS correctness.